Structure-Oncall/Period.java

```diff
@@ -0,0 +1,16 @@
+public class period{
+	String course;
+	String teachable;
+	int roomNumber;
+	int periodNumber;
+	boolean absent=false;
+	public period(boolean isAbsent){
+		absent=true;
+	}
+	public period()

```

Comment:
Shouldn't the Period constructors take course and teachable and all of the other instance variables as parameters, especially since there are no mutator methods. To that end, there should probably be a toggleAbsent or setAbsent method since its the one thing that may change.